src/main/kotlin/org/jetbrains/kotlinx/jupyter/apiImpl.kt

```diff
@@ -149,6 +152,11 @@ class NotebookImpl(
     override val jreInfo: JREInfoProvider
         get() = JavaRuntime
 
+    fun updateVariablesState(evaluator: InternalEvaluator) {
```

Comment:
It seems like a very internal method. We most likely don't want to expose it to user